cmd/cli/main.go

```diff
@@ -12,8 +13,11 @@ import (
 	"os"
 	"regexp"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	"strings"
 )
 
+//curl 'https://rhcc-api.redhat.com/rest/v1/repository/registry.access.redhat.com/amq7%252Famq-online-1-api-server/images' -H 'accept: application/json' -H 'Origin: https://access.redhat.com' -H 'Sec-Fetch-Mode: cors' --compressed
```

Comment:
@maleck13 re this API - any idea how "supported" this is? is this the one from the RH security api, or an API powering the portal? Mostly just curious if it'll a) break often or b) go away at some point and inconvenience us

Reply:
I don't know who owns this api. Would like to find out though.